tests/test_web/test_broker_error.py

```diff
@@ -0,0 +1,116 @@
+from unittest import mock
+
+from kombu.exceptions import OperationalError
+
+from iib.web.models import Request, RequestStateMapping, RequestState
+
+
+def assert_testing(rv, mock_smfsc, db):
+    response_json = {'error': "The scheduling of the build request with ID 2 failed."}
+    assert rv.status_code == 500
+    assert response_json == rv.json
+    assert mock_smfsc.call_count == 2
+
+    req_state = db.session.query(Request).filter(Request.id == 2).one()
```

Comment:
```suggestion
    req_state = db.session.query(Request).get(2)
```

Reply:
This needs to be getting the request with the ID of `1`. Since your test starts with a clean database, there should be only the request that you created as part of the test.